### ISSUE TITLE
add salt download progress indicator

### ISSUE THREAD

**Issue by adrelanos** [QubesOS/qubes-issues#4215]
### Qubes OS version:
any
### Affected component(s): 
dom0

---

### Steps to reproduce the behavior:
For example Whonix installation.

https://www.whonix.org/wiki/Qubes/Install

    sudo qubesctl state.sls qvm.anon-whonix

But any template installed through salt should be affected.

### Expected behavior:
Progress indicator during salt download and salt processing.

### Actual behavior:
No progress indicator during salt download.

### General notes:
This is a grave usability issue. There may be no feedback for hours since downloads are slow depending on network connection (and mirror speed perhaps). Only very technical users can check if download is still in progress or the process is just hanging.

https://groups.google.com/d/msg/qubes-users/fwCqxENXguY/jYnQVLq3AwAJ

### Related issues:
This issue will be more and more relevant in future for any template due to https://github.com/QubesOS/qubes-issues/issues/3447.

**Comment by averyfuentes9rs** [comment]
Following an older discussion in [1], it seems the 'use_vt' parameter of certain states like 'cmd.run' could solve this on the salt side of things. 
For the qubes side of things, I lack understanding if this is possible. If so, I would love to see this happening myself.

[1] https://groups.google.com/forum/#!topic/salt-users/ikAVtZnuB30
[2] https://docs.saltstack.com/en/latest/ref/states/all/salt.states.cmd.html#salt.states.cmd.run

**Comment by ben-grande** [comment]
This is up to Salt... Ansible has. For Salt, you can end the command-line with `-l debug`.

**Comment by adrelanos** [comment]
`-l debug` shows a massive amount of output and may therefore not be suitable as default?

**Comment by ben-grande** [comment]
> `-l debug` shows a massive amount of output and may therefore not be suitable as default?

Yes, not suitable as default, but that is a Salt issue, they only show relevant information when "DEBUG" is used.
